websockets/framing.py

```diff
@@ -14,10 +14,18 @@
 import io
 import random
 import struct
+from itertools import cycle
 
 from .exceptions import PayloadTooBig, WebSocketProtocolError
 
 
+try:
+    from .cext import apply_mask
+except ImportError:
+    def apply_mask(mask, data):
```

Comment:
I will get confused forever if the signature is `(mask, data)` rather than `(data, mask)`. I'm not sure why but I really expect the arguments to be the other way round.